cmd/deploy/cfghandler.go

```diff
@@ -29,6 +29,7 @@ import (
 type configMapHandler interface {
 	translateConfigMapAndDeploy(context.Context, *pipeline.CfgData) (*apiv1.ConfigMap, error)
 	updateConfigMap(context.Context, *apiv1.ConfigMap, *pipeline.CfgData, error) error
+	updateEnvs(context.Context, string, string, []string) error
```

Comment:
Can we be more explicit? 
```suggestion
	updateEnvsFromCommands(context.Context, string, string, []string) error
```

Reply:
done f3b410440639b89a763e32b632327c3fd024fbc9